PeerioTechnologies/peerio-desktop
# feat: room invite notifications

#### Relevant info and issue/PR links 

* https://app.clubhouse.io/peerio/story/3738/desktop-notification-for-new-rooms
* https://github.com/PeerioTechnologies/peerio-icebear/pull/170 

#### Testing instructions  

1. Launch two apps with different accounts.
2. Invite the other account and keep focus away from the app with invited account.
3. *Will receive notification about this invite.*
4. Click on notification.
5. *The app will be brought to front and open the invite.*
6. Quit the app.
7. Re-launch the app with account that was invited and verify that there's no notifications about new invites after launching.
8. Go to Preferences, turn off invite notifications (Desktop notifications > You are invited to a room). Verify that there's no notification this time on step 3 when repeating from step 1.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
